library/common/jni/android_jni_utility.cc

```diff
@@ -33,3 +33,17 @@ bool is_cleartext_permitted(absl::string_view hostname) {
   return true;
 #endif
 }
+
+void tag_socket(int ifd, int uid, int tag) {
+#if defined(__ANDROID_API__)
+  JNIEnv* env = get_env();
+  jclass jcls_AndroidNetworkLibrary = env->FindClass("org/chromium/net/AndroidNetworkLibrary");
```

Comment:
The one thing that you need to do here is to start using the `find_class` that was introduced in https://github.com/envoyproxy/envoy-mobile/pull/2483

```suggestion
  jclass jcls_AndroidNetworkLibrary = find_class("org.chromium.net.AndroidNetworkLibrary");
```